src/main/java/seedu/address/model/consultation/Consultation.java

```diff
@@ -0,0 +1,104 @@
+package seedu.address.model.consultation;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import seedu.address.model.student.Student;
+
+/**
+ * Represents a Consultation in the system.
+ * Guarantees: details are present and not null, field values are validated.
+ */
+public class Consultation {
+
+    // Identity fields
+    private final Date date;
+    private final Time time;
+
+    // Data field
+    private final List<Student> students;
+
+    /**
+     * Every field must be present and not null.
+     * The list of students can be empty but not null.
+     */
```

Comment:
Should we standardize to follow JavaDocs Documentation? e.g. @param, @returns, @throws. Same Issue for the other Documentation.

Reply:
Seconded. Also, my IntelliJ automatically includes appropriate @param/@returns/@throws after I type `/**` and hit enter, so it should be possible to get your IntelliJ to do this too somehow.